.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": true
+  },
+  "editor.formatOnSave": true,
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[json]": {
```

Comment:
Just a question where can I find the documentation for all these setting options?

Reply:
Look into stuff related to `setting.json` for VSCode. This is a good page: https://code.visualstudio.com/docs/getstarted/settings